lib/SetAPI/rnaseq_set_functions.py

```diff
@@ -0,0 +1,128 @@
+"""Functions for retrieving various RNASeq sets from the DB."""
+from typing import Any
+
+from installed_clients.WorkspaceClient import Workspace
+
+from SetAPI.util import populate_item_object_ref_paths
+
+
+def get_rnaseq_set(
+    workspace_client: Workspace,
+    obj_selector: dict[str, str],
+    set_type: str,
+    obj_data: dict[str, Any],
+    include_item_info: bool = False,
+    include_set_item_ref_paths: bool = False,
+) -> dict[str, Any]:
+    """Retrieve and reconstitute a KBase RNASeqAlignmentSet or RNASeqExpressionSet from the workspace.
```

Comment:
these two methods were almost identical - only difference is the keys used for the list data and the map data

Reply:
apart from the alteration to use a variable for the list_data_key / map_data_key, the rest is c&p'd